VideoLocker/src/main/java/org/edx/mobile/view/VideoListActivity.java

```diff
@@ -195,10 +195,7 @@ public void run() {
             }
 
             String filepath = null;
-            // check if file available on local
-            if (video.isVideoForWebOnly) {
```

Comment:
I see that it's a warning, but I believe the change you added changes the logic.